### PR TITLE
WT-6083 Enable checkpoint-test and fix __wt_txn_set_commit_timestamp failure

### PR DESCRIPTION
**Pull request by raviprakashgiri29**
The Evergreen patch build link

https://evergreen.mongodb.com/task/wiredtiger_ubuntu1804_checkpoint_test_patch_7812adc84fee863cb2b1f594938f18a28572aa2b_5eba2af2d6d80a69d4cb00d6_20_05_12_04_50_13

### CODE REVIEW

**Comment by tammybailey** [comment]
@raviprakashgiri29 You'll also need to remove the comments in `./test/checkpoint/Makefile.am` for the test to compile and run.

**Comment by raviprakashgiri29** [comment]
> Agree with Tammy - when disabling/enabling pull request tasks, there are 2 places to be covered (evergreen.yml and makefile).

@lukech Yeah, I agree. I missed updating the Makefile.am. Fixed it now.

**Comment by raviprakashgiri29** [comment]
> @raviprakashgiri29 You'll also need to remove the comments in `./test/checkpoint/Makefile.am` for the test to compile and run.

@tammybailey Yeah, I missed updating the Makefile.am. Fixed it now.

**Comment by raviprakashgiri29** [comment]
Closing this PR as of now. I will reopen it with the fix.

**Comment by raviprakashgiri29** [comment]
Temporarily disabled LSM, column-store, mixed tables, and Sweep /timestamps checks

File: test/checkpoint/smoke.sh [hunk 1]
```diff
@@ -5,33 +5,41 @@ set -e
...
 $TEST_WRAPPER ./t -T 6 -t r
 
 echo "checkpoint: 6 row-store tables, named checkpoint"
 $TEST_WRAPPER ./t -c 'TeSt' -T 6 -t r
 
-echo "checkpoint: row-store tables, stress history store. Sweep and timestamps"
-$TEST_WRAPPER ./t -t r -W 3 -r 2 -D -s -x -n 100000 -k 100000 -C cache_size=100MB
+# Temporarily disabled
+#echo "checkpoint: row-store tables, stress history store. Sweep and timestamps"
+#$TEST_WRAPPER ./t -t r -W 3 -r 2 -D -s -x -n 100000 -k 100000 -C cache_size=100MB
```
Comment (sueloverso) [inline test/checkpoint/smoke.sh:33]: This is relevant to MongoDB I believe. We should be testing this.

File: test/checkpoint/smoke.sh [hunk 2]
```diff
@@ -5,33 +5,41 @@ set -e
...
 
-echo "checkpoint: row-store tables, Sweep and timestamps"
-$TEST_WRAPPER ./t -t r -W 3 -r 2 -s -x -n 100000 -k 100000 -C cache_size=100MB
+# Temporarily disabled
+#echo "checkpoint: row-store tables, Sweep and timestamps"
+#$TEST_WRAPPER ./t -t r -W 3 -r 2 -s -x -n 100000 -k 100000 -C cache_size=100MB
```
Comment (sueloverso) [inline test/checkpoint/smoke.sh:37]: This is also relevant to MongoDB I believe.

File: test/checkpoint/smoke.sh (hunk 1 above)
Comment (agorrod) [reply test/checkpoint/smoke.sh:33]: I put something in the JIRA ticket.

**Comment by raviprakashgiri29** [comment]
Thank you @agorrod for clarifying the reason for disabling some of the tests.

@sueloverso Can you please approve this PR?

**Pull request merged**